fundamentals/00_overview.ipynb

```diff
@@ -1,196 +0,0 @@
-    "- [Code Repository](https://github.com/pydata/xarray)\n",
-    "\n",
-    "1. Ask for help:\n",
-    "\n",
-    "- Use the\n",
-    "  [python-xarray](https://stackoverflow.com/questions/tagged/python-xarray) on\n",
-    "  StackOverflow\n",
-    "- [GitHub Issues](https://github.com/pydata/xarray/issues) for bug reports and\n",
-    "  feature requests\n",
-    "\n",
-    "## Tutorial Structure\n",
-    "\n",
-    "This tutorial is made up of multiple Jupyter Notebooks. These notebooks mix\n",
-    "code, text, visualization, and exercises.\n",
-    "\n",
-    "If you haven't used JupyterLab before, it's similar to the Jupyter Notebook. If\n",
-    "you haven't used the Notebook, the quick intro is\n",
-    "\n",
-    "1. There are two modes: command and edit\n",
-    "1. From command mode, press Enter to edit a cell (like this markdown cell)\n",
-    "1. From edit mode, press Esc to change to command mode\n",
-    "1. Press shift+enter to execute a cell and move to the next cell.\n",
-    "1. The toolbar has commands for executing, converting, and creating cells.\n",
-    "\n",
-    "The layout of the tutorial will be as follows:\n",
-    "\n",
-    "1. [Introduction + data structures for multi-dimensional data](./01_datastructures_and_io.ipynb)\n",
-    "1. [Working with labeled data](02.1_working_with_labeled_data.ipynb)\n",
-    "1. [Manipulating dimensions](02.2_manipulating_dimensions.ipynb)\n",
-    "1. [Aligning data objects](02.3_aligning_data_objects.ipynb)\n",
-    "1. [Computation with Xarray](03.1_computation_with_xarray.ipynb)\n",
-    "1. [Groupby with Xarray](03.2_groupby_with_xarray.ipynb)\n",
-    "1. [Plotting and visualization](04_plotting_and_visualization.ipynb)\n",
-    "1. [Introduction to Dask](05_intro_to_dask.ipynb)\n",
-    "1. [Dask and Xarray](06_xarray_and_dask.ipynb)\n",
-    "\n",
-    "## Exercise: Print Hello, world!\n",
-    "\n",
-    "Each notebook will have exercises for you to solve. You'll be given a blank or\n",
-    "partially completed cell, followed by a hidden cell with a solution. For\n",
-    "example.\n",
-    "\n",
-    "Print the text \"Hello, world!\".\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Your code here"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "In some cases, the next cell will have the solution. Click the ellipses to\n",
-    "expand the solution, and always make sure to run the solution cell, in case\n",
-    "later sections of the notebook depend on the output from the solution.\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "jupyter": {
-     "source_hidden": true
-    }
-   },
-   "outputs": [],
-   "source": [
-    "print(\"Hello, world!\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Going Deeper\n",
```

Comment:
I think this is valuable and could go as a separate heading under "Overview".

Reply:
good catch i forgot to copy that over. i'll update the links to the repositories b/c pangeo binder is still broken